package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "precise-ui",
-  "version": "0.6.3",
+  "version": "0.6.4",
```

Comment:
No need for bumping up the version. We released nothing new after `0.6.1`. I would leave `0.6.2` and update CHANGELOG with respective changes.

Reply:
The previous version was 0.6.3, I would not prefer to downgrade version. Also, the current Pharos dependency version is 
![image](https://user-images.githubusercontent.com/11165501/62135088-e45a0000-b2e1-11e9-9355-af9de09c8099.png)
therefore I prefer to bump version.

Reply:
yes, looks like you've introduced `0.6.3` as well 😏

Reply:
I suggest discussing this with main maintainers @kotielnikov @FlorianRappl @boromo

Reply:
The Semver Standart says: Bug fixes not affecting the API increment the patch version. 0.6.3 -> 0.6.4 is the patch version incrementing.

Reply:
1) there is no need to increase the version, it's already higher than on `master`. 
2) For prerelease version (anything lower than 1.0.0) minor becomes major and patch becomes minor.

Reply:
> Patch version Z (x.y.Z | x > 0) MUST be incremented if only backwards compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.
@IKolosynskyi your fix is definitely not a BC